docs/using.rst

```diff
@@ -359,21 +359,21 @@ By default, the following locations are searched:
 Getting help
 ============
 
-If you're having problems you can chat with us on `IRC (#letsencrypt @
-Freenode) <https://webchat.freenode.net?channels=%23letsencrypt>`_ or
-get support on our `forums <https://community.letsencrypt.org>`_.
+If you're having problems you can chat with us on `IRC (#certbot @
+OFTC) <https://webchat.oftc.net?channels=%23certbot>`_ or
```

Comment:
Are we moving to OFTC? I'm fine either way, however, @pde and I are hanging out in #certbot on Freenode.